src/gabc/gabc-write.c

```diff
@@ -231,7 +231,7 @@ static void gabc_print_char(FILE *f, const grewchar to_print)
 {
     if (write_state == GABC_AT_PROTRUSION_FACTOR) {
         write_state = GABC_IN_PROTRUSION_FACTOR;
-        if (to_print == ':') {
+        if (to_print == 'd') {
```

Comment:
I admit I don't get it... can you point me to the line in the source that sets something to 'd'? (this sounds like an uninitialized variable?)